FeedStoreChallenge/RealmObjects/FeedRealmObject.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  FeedRealmObject.swift
+//  FeedStoreChallenge
+//
+//  Created by Пильтенко Роман on 13.02.2021.
+//  Copyright © 2021 Essential Developer. All rights reserved.
+//
+
+import Foundation
+import RealmSwift
+
+final class FeedRealmObject: Object {
+	@objc dynamic var objID: ObjectId = ObjectId()
+	@objc dynamic var timestamp: Date = Date.init()
```

Comment:
The type definition can be simplified:

```suggestion
	@objc dynamic var timestamp = Date()
```